src/_guides/technical-writing/the-role-of-technical-writer-in-api-documentation.md

```diff
@@ -24,6 +24,8 @@ For technical writers who have a software developer background, the transition t
 
 Technical writers also benefit from learning how to use API client tools, such as Postman, Insomnia, and Hoppscotch. Command-line tools such as cURL may also be useful but aren’t always necessary as it depends on the preferences of the intended audience that will use the API.
 
+> Bump.sh now includes its own [API Explorer](https://docs.bump.sh/product-updates/2025/01/09/api-explorer/), allowing you to test your API directly within your documentation. [Give it a try !](https://bump.sh/demo/doc/api-explorer/explorer/operation/operation-adduser)
+
```

Comment:
```suggestion
```

Although this can be valuable information for our readers, I would refrain from adding it in this specific text, which focuses on Technical Writers skills rather than tooling.